systemtest/src/test/java/io/strimzi/systemtest/mirrormaker/MirrorMaker2ST.java

```diff
@@ -1069,6 +1075,499 @@ void testRestoreOffsetsInConsumerGroup(ExtensionContext extensionContext) {
+            .build();
+        kubeClient().patchSecret(testStorage.getNamespaceName(), kafkaUserSourceName, passwordSource);
+        mmSnapshot = DeploymentUtils.waitTillDepHasRolled(testStorage.getNamespaceName(), KafkaMirrorMaker2Resources.deploymentName(testStorage.getClusterName()), 1, mmSnapshot);
+
+        LOGGER.info("[SCRAM-SHA] Changing KafkaUser sha-password on KMM2 Target");
+        Secret passwordTarget = new SecretBuilder()
+            .withNewMetadata()
+            .withName(kafkaUserTargetName)
+            .endMetadata()
+            .addToData("password", "dGFyZ2V0LXBhc3N3b3Jk")
+            .build();
+        kubeClient().patchSecret(testStorage.getNamespaceName(), kafkaUserTargetName, passwordTarget);
+        mmSnapshot = DeploymentUtils.waitTillDepHasRolled(testStorage.getNamespaceName(), KafkaMirrorMaker2Resources.deploymentName(testStorage.getClusterName()), 1, mmSnapshot);
+
+        LOGGER.info("[SCRAM-SHA] Recreate kafkaClients pod with new passwords.");
+        resourceManager.deleteResource(kubeClient().namespace(testStorage.getNamespaceName()).getDeployment(testStorage.getKafkaClientsName()));
+        resourceManager.createResource(extensionContext, KafkaClientsTemplates.kafkaClients(testStorage.getNamespaceName(), true, testStorage.getKafkaClientsName(), userSource, userTarget).build());
+        kafkaClientsPodName = kubeClient().listPodsByPrefixInName(testStorage.getKafkaClientsName()).get(0).getMetadata().getName();
+
+        internalKafkaClient = internalKafkaClient.toBuilder()
+            .withUsingPodName(kafkaClientsPodName)
+            .withTopicName(topicSourceNameB)
+            .withClusterName(kafkaClusterSourceName)
+            .withKafkaUsername(userSource.getMetadata().getName())
+            .withListenerName(kafkaTlsScramListenerName)
+            .build();
+
+        LOGGER.info("[SCRAM-SHA] Sending messages to - topic {}, cluster {} and message count of {}.",
+            topicSourceNameB, kafkaClusterSourceName, messagesCount);
+        sent = internalKafkaClient.sendMessagesTls();
+
+        LOGGER.info("[SCRAM-SHA] Changing to target - topic {}, cluster {}, user {} for next internalKafkaClient.",
+                topicTargetNameB, kafkaClusterTargetName, userTarget.getMetadata().getName());
+        internalKafkaClient = internalKafkaClient.toBuilder()
+            .withTopicName(topicTargetNameB)
+            .withClusterName(kafkaClusterTargetName)
+            .withKafkaUsername(userTarget.getMetadata().getName())
+            .withConsumerGroupName(ClientUtils.generateRandomConsumerGroup())
+            .build();
+
+        LOGGER.info("[SCRAM-SHA] Now messages should be mirrored to target topic and cluster");
+        internalKafkaClient.consumesTlsMessagesUntilOperationIsSuccessful(sent, Duration.ofMinutes(10).toMillis());
+        LOGGER.info("[SCRAM-SHA] Messages successfully mirrored");
+    }
+
+    @ParallelNamespaceTest
+    @Tag(ACCEPTANCE)
```

Comment:
Same as above

Reply:
ok, I will move it to regression tag then